pkg/auth/service.go

```diff
@@ -2197,3 +1145,68 @@ func NewAPIAuthServiceWithClient(client ClientWithResponsesInterface, externalPr
 		externalPrincipalsEnabled: externalPrincipalseEnabled,
 	}, nil
 }
+
+func CheckPermissions(ctx context.Context, node permissions.Node, username string, policies []*model.Policy) CheckResult {
```

Comment:
Again - git marks this function as brand new and it was only moved per my understand. Let me know if a deeper review is needed here.

Reply:
Yeah, I needed to export this function since acl service is using it too. That's why it shows a change. But the code remained the same